lib/addons/ssm-agent/ssm-install.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+echo '* * * * * root yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm && rm -rf /etc/cron.d/ssmstart' > /etc/cron.d/ssmstart
```

Comment:
Can we point to an explicit version of the agent and tag the image with that version, so that we could have deterministic upgrades? The use case that I have in mind: we will upgrade the agent and publish it implicitly as latest. Customers will pick it up immediately so that in the existing cluster the SSP will be running a different version of the agent with the same SSP version. It is good when the agent works flawlessly, but it is great if one of the versions fails for any reason and we can't reproduce the state.

Reply:
addressed